src/service/services/UserService.ts

```diff
@@ -67,6 +67,13 @@ export class UserService {
         user.password,
         await bcrypt.genSalt(JWT_CONFIG.salt)
       );
+      //Validate user existence
+      const userExists = await this.userRepository.count({
```

Comment:
Creo que este es un nombre de variable más acorde a lo que representa

```suggestion
      const foundUserCount = await this.userRepository.count({
```